lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.html

```diff
@@ -11,7 +7,8 @@
             data-automation-class="read-only-value">{{ (property.displayValue | async) | translate }}
         </div>
         <div *ngIf="isEditable()">
-            <mat-form-field class="adf-select-item-padding-editable adf-property-value">
+            <mat-form-field class="adf-property-value" appearance="standard">
+                <mat-label>{{ property.label }}</mat-label>
```

Comment:
Isn't it the breaking change for some use cases? Right now the label will only be displayed when the field is editable

Reply:
Re-added the label only for non-editable mode. For editable mode, we are still using the mat-label introduced in this PR. The reason I went for this approach is to maintain consistency of this component with the rest of the applications. I've also added corresponding unit tests for both the labels